src/block/block_open.c

```diff
@@ -214,9 +196,6 @@ __wt_block_open(WT_SESSION_IMPL *session, const char *filename, uint32_t objecti
     WT_ERR(__wt_calloc_one(session, &block));
     WT_ERR(__wt_strdup(session, filename, &block->name));
     block->objectid = objectid;
-    block->ref = 1;
```

Comment:
It looks like this initialization of the reference count is missing in the new version. Or did I miss it?

Reply:
No accidental edit. Reverted.